src/client/settings/org-webhook-list.ts

```diff
@@ -28,22 +28,35 @@ async function getState() {
 }
 
 function hookTemplate(org: OrgWebHookState) {
-  // TODO: No idea how to add disabled state (i.e. bring
-  // org.viewerCanAdminister to checkbox)
+  const checkboxClick = () => {
+    console.log('Checkbox Click: ', org);
+  };
   return html`
   <div class="settings-toggle-item">
-    <input class="settings-toggle-item__toggle" type="checkbox"></input>
+    <input class="settings-toggle-item__toggle" type="checkbox" on-click="${() => checkboxClick()}"></input>
```

Comment:
Side comment: I wonder if we want to use a button like "Set up" instead of a toggle. Not sure we really need to support turning it off on our end - they can uninstall via GitHub.

Reply:
This is toggling the org webhook so we could just have setup and change - I'm just copying the mocks at the moment and personally I'm a fan of - "The place you set up is the place you turn it off". Setting up and then leaving it is kinda of a pain when I've used sites that do it.